assets/charts/components/azure-arc-lokomotive/templates/rbac.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: azure-arc-onboarding
```

Comment:
This role can be part of the hook.

Reply:
This might  create an issue when executing `lokoctl component delete` and not finding the ClusterRoleBinding.

Reply:
It should be fine this to be part of both hooks. But this is not needed during cluster operation.

Reply:
done.

Reply:
Update: when adding hooks 

Upon removal of the component, I see that clusterrole and clusterrolebinding are still present.
If there are no hooks added, the resources get cleaned up after installation.

Reply:
Maybe try something like:
```
    "helm.sh/hook": pre-install,pre-upgrade,post-install,post-upgrade
    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
```

This is what webhook certificate injector is using, not sure if this is all needed.

Reply:
@invidian I am still not sure about this

I don't understand what you meant by this comment https://github.com/kinvolk/lokomotive/pull/1473#discussion_r648978205

What do you mean by cluster operation ? How it interferes during cluster operation ?

Without hook, I see no issues.

Reply:
This role is only needed when the hook is running, so during installation and uninstallation of the chart. After Helm is finished, it should be fine to remove those, right? This is what hook is suppose to do.

Reply:
updated. PTAL